assets/sass/protocols/_quiz1.scss

```diff
@@ -0,0 +1,74 @@
+@import '../components/icons.scss';
```

Comment:
The `@use` might suit this better https://sass-lang.com/documentation/at-rules/use